scss/grouparoo.scss

```diff
@@ -11,7 +11,7 @@ $grouparoo-blue: #29abe2;
 $grouparoo-light-blue: #b3e8ff;
 $grouparoo-blue-gray: #9b9db1;
 $grouparoo-violet: #9954bb;
-$grouparoo-light-violet: #dfe8fb;
+$grouparoo-light-violet: #d3d3ff;
```

Comment:
I think we shouldn't do this, and instead do:
`$info: $grouparoo-violet;`

Reply:
Sounds good.  That improves the contrast over the old badge color as well.

![Screen Shot 2021-12-06 at 9 28 53 AM](https://user-images.githubusercontent.com/63751206/144863742-b15408b5-001e-4ca5-b571-c88977df9ed0.png)